serf/serf.go

```diff
@@ -25,7 +25,7 @@ import (
 // version to memberlist below.
 const (
 	ProtocolVersionMin uint8 = 2
-	ProtocolVersionMax       = 4
+	ProtocolVersionMax       = 5
```

Comment:
It looks consul is on the latest serf version already, so it seems like this shouldn't be an issue: https://github.com/hashicorp/consul/blob/master/consul/config.go#L41-L47